### PR TITLE
[docs] add description to create node-group-token

### PR DESCRIPTION
**Pull request by kirill-kashin**
## Description

Description to create node-group-token for add a batch of static nodes to a cluster

## Why do we need it, and what problem does it solve?

In k8s >= 1.25 version refused to create tokens for service accounts by default. And the documentation suggested this case just now

## What is the expected result?

Users will no longer have to ask questions about where the token went

```changes
section: node-manager docs
type: fix 
impact_level: low
```

### CODE REVIEW

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @z9r5. I've started the workflow for label 'deploy/web/stage'...

File: modules/040-node-manager/docs/FAQ.md [hunk 1]
```diff
@@ -52,14 +52,20 @@ You can automate the bootstrap process with any automation platform you prefer.
    kubectl get ep kubernetes -o json | jq '.subsets[0].addresses[0].ip + ":" + (.subsets[0].ports[0].port | tostring)' -r
    ```
 
-2. Get Kubernetes API token for special `ServiceAccount` that Deckhouse manages:
+2. Check the k8s version if it is >= 1.25, you should create ``node-group-token``.
```
Comment (VV-corrector) [inline modules/040-node-manager/docs/FAQ.md]: K8s

File: modules/040-node-manager/docs/FAQ.md [hunk 2]
```diff
@@ -52,14 +52,20 @@ You can automate the bootstrap process with any automation platform you prefer.
    kubectl get ep kubernetes -o json | jq '.subsets[0].addresses[0].ip + ":" + (.subsets[0].ports[0].port | tostring)' -r
    ```
 
-2. Get Kubernetes API token for special `ServiceAccount` that Deckhouse manages:
+1. Check the k8s version if it is >= 1.25, you should create ``node-group-token``.
```
Comment (VV-corrector) [inline modules/040-node-manager/docs/FAQ.md:55]:
```suggestion
1. Check the K8s version if it is >= 1.25, you should create ``node-group-token``.
```

File: modules/040-node-manager/docs/FAQ_RU.md [hunk 3]
```diff
@@ -52,14 +52,20 @@ search: добавить ноду в кластер, добавить узел 
    kubectl get ep kubernetes -o json | jq '.subsets[0].addresses[0].ip + ":" + (.subsets[0].ports[0].port | tostring)' -r
    ```
 
-2. Получите Kubernetes API-токен для специального `ServiceAccount`, которым управляет Deckhouse:
+1. Проверьте версию k8s если она >= 1.25, то следует создать `node-group-token`.
```
Comment (VV-corrector) [inline modules/040-node-manager/docs/FAQ_RU.md:55]:
```suggestion
1. Проверьте версию K8s, если она >= 1.25, следует создать `node-group-token`.
```

File: modules/040-node-manager/docs/FAQ_RU.md [hunk 4]
```diff
@@ -97,7 +103,7 @@ search: добавить ноду в кластер, добавить узел 
          when: bootstrapped.stat.exists == False
    ```
 
-4. Определите дополнительную переменную `node_group`. Значение переменной должно совпадать с именем `NodeGroup`, которой будет принадлежать узел. Переменную можно передать различными способами, например с использованием inventory-файла:
+!. Определите дополнительную переменную `node_group`. Значение переменной должно совпадать с именем `NodeGroup`, которой будет принадлежать узел. Переменную можно передать различными способами, например с использованием inventory-файла:
```
[the remark this answers is not in the document]
Comment (z9r5) [reply modules/040-node-manager/docs/FAQ_RU.md:106]: Да. Ошибка.
Спасибо.

Comment (z9r5) [inline modules/040-node-manager/docs/FAQ_RU.md:106]:
```suggestion
1. Определите дополнительную переменную `node_group`. Значение переменной должно совпадать с именем `NodeGroup`, которой будет принадлежать узел. Переменную можно передать различными способами, например с использованием inventory-файла:
```

**Comment by deckhouse-BOaTswain** [comment]
Cherry pick PR [6036](https://github.com/deckhouse/deckhouse/pull/6036) to the branch [release-1.52](https://github.com/deckhouse/deckhouse/tree/release-1.52) successful!

**Pull request merged**